synfig-studio/src/gui/render.cpp

```diff
@@ -576,6 +577,17 @@ RenderSettings::on_finished(std::string error_message)
 			App::sound_render_done->set_position(Time());
 			App::sound_render_done->set_playing(true);
 		}
+
+		const filesystem::Path filename(entry_filename.get_text());
+		std::vector<std::string> ext_mult = {{".bmp"}, {".png"}, {".jpg"}, {".exr"}, {".ppm"}};		
```

Comment:
Please use a better name for the variable/constant that describes what it stores.